src/main/java/org/apache/sysml/runtime/matrix/data/LibMatrixDatagen.java

```diff
@@ -510,14 +511,19 @@ private static void genRandomNumbers(boolean invokedFromCP, int rl, int ru, int
 
 		// Setup Pseudo Random Number Generator for cell values based on 'pdf'.
 		if (valuePRNG == null) {
-			if ( rgen._pdf.equalsIgnoreCase(RAND_PDF_UNIFORM)) 
+			switch (rgen._pdf) {
+			case UNIFORM:
```

Comment:
hmm, thanks for pointing this out; apparently I missed this PR somehow - then we need to have a broader discussion regarding the style template in general. Regarding the switch statement it's probably safe to use indentation here because it significantly increases readability.